### ISSUE TITLE
Getting rid of previous values.

### ISSUE THREAD

**Issue by TraceurColin** [sproutsocial/inputosaurus-text#6]
I'm building a project that allows the user to create nodes using jstree, I've used this to help for multiple inputs at once. I've got the code to write the inputted nodes and to even to get rid of the previous items in the list so they don't show up when the text box re creates its self. My issue is that it retains the old inputs and never gets rid of them so as you type the next word it brings back the old stuff, is there any way to remove them?

**Comment by dansnetwork** [comment]
Are you able to put your forked code up in a repo where I can take a look?

**Comment by TraceurColin** [comment]
I'll try to put it up. In the mean time, I'm referencing the inputosaurus as a library could this be a problem? and in the inputosaurus I've only changed one line of code and that is in the _createTag function I changed

"return ... data-inputosaurus="' +name+ '"class>...

to

"return ... ' data-inputosaurus="' +name+ '"class = datasaurus>...

so I could run a $("li.datasaurus").remove(); to get rid of all of the list items in my jstree code. (the code that references the inputosaurus.js)

Is there an array that contains the previously typed in selections that I could empty?
